src/remutex.rs

```diff
@@ -18,6 +18,12 @@ unsafe impl GetThreadId for RawThreadId {
     fn nonzero_thread_id(&self) -> usize {
         // The address of a thread-local variable is guaranteed to be unique to the
         // current thread, and is also guaranteed to be non-zero.
+        //
+        // REVIEW: is the usage of `mem::uninitialized` really justified here?
```

Comment:
should use `MaybeUninit`?

Reply:
That's missing the point, it's literally a byte where the constant 0 can be used and remove unnecessary unsafe.